app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,3 @@
+class EntitiesController < ApplicationController
+  before_action :authenticate_user!
+end
```

Comment:
@HFG43 [OPTIONAL] I kindly suggest you rename this controller to align it with the name of the **Expense** model